Simplenote/src/main/java/com/automattic/simplenote/NotesActivity.java

```diff
@@ -648,16 +652,30 @@ private void checkForSharedContent(Intent intent) {
         }
     }
 
+    private List<Tag> getTagsFromCursor(Bucket.ObjectCursor<Tag> tagCursor) {
+        List<Tag> tags = new ArrayList<>();
+
+        for (int i = 0; i < tagCursor.getCount(); i++) {
+            tagCursor.moveToNext();
+            Tag tag = tagCursor.getObject();
+            if (!collaboratorsRepository.isValidCollaborator(tag.getName())) {
```

Comment:
I'm curious about this. Since this is really just a check for a valid email address, then if a user previously used an email address as a tag it won't show up any more. Is that anything to be concerned about?

Reply:
If the tag (email) is assigned to a note, it can be accessed from the collaborators' UI. If it is not assigned to a note, then the user would not be able to access it. The concern here would be that there are some tags (emails) that should be deleted but there won't be a way to delete them. Emails will be accessible just from the collaborators' UI for each note.